ui/src/api/sampleChanger.js

```diff
@@ -30,10 +30,6 @@ export function sendMountSample(sampleData) {
   return endpoint.post(sampleData, '/mount').res();
 }
 
-export function sendUnmountSample(sample) {
-  return endpoint.post({ sample: { location: sample } }, '/unmount').res();
```

Comment:
So the issue you had @elmjag was here, `sample` is an object that contains `location` so this logic simply wrapped that object in a object with the key `location` that on the server side gets passed as a `dict` instead of the location `str`